runtime/parser2/errors.go

```diff
@@ -211,6 +226,8 @@ var _ ParseError = &MissingCommaInParameterListError{}
 
 func (*MissingCommaInParameterListError) isParseError() {}
 
+func (e *MissingCommaInParameterListError) IsUserError() {}
```

Comment:
Also, maybe add a type assertion like for the other parse errors above.

```suggestion
func (*MissingCommaInParameterListError) IsUserError() {}
```